### ISSUE TITLE
Fix Wikipedia link by escaping brackets

### ISSUE THREAD

**Issue by DenVitko** [qiskit-community/qiskit-textbook#1453]
### Describe the problem

See the page about density matrix, [part Trace and Positivity Conditions](https://qiskit.org/textbook/ch-quantum-hardware/density-matrix.html#Trace-and-Positivity-Conditions-)

---
Its trace must be equal to one:

Remembering that the [trace of a matrix](https://en.wikipedia.org/wiki/Trace_(linear_algebra)) (denoted as Tr) is the sum of its diagonal terms, we have:

### Describe the expected solution

Try to escape the brackets with `\`, or see other options [here](https://stackoverflow.com/questions/13824669/how-do-you-write-a-link-containing-a-closing-bracket-in-markdown-syntax).

---
I'd do it, but wasn't sure how to make a push request. :/

**Comment by divshacker** [comment]
Hey @DenVitko
Thanks for reporting this. 
Making a pull request is really easy. When you open a [Github repo](https://github.com/qiskit-community/qiskit-textbook) of qiskit textbook You will see a option of forking it. Just fork the repo. Make a new branch then change the files to fix this issue and then commit that then open a pull request. It's very easy. 
[This](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) might help you.

**Comment by Anantha-Rao12** [comment]
Hi @divshacker @DenVitko 
I am unsure if the issue is with the text on the corresponding jupyter notebook. When I load the notebook locally, the extra bracket is absent. Not sure what is causing the issue on the textbook online. 

![image](https://user-images.githubusercontent.com/48722343/188841386-06e7b95b-14e6-4e27-a459-26bdefe5ae25.png)

**Comment by divshacker** [comment]
Yeah I have also tried to fix this issue but faced same issue which @Anantha-Rao12 is facing right now. Running the notebook locally don't give this error and all links are working. 
I think there must be some internal rendering problem. 
@frankharkins any suggestions?

**Comment by DenVitko** [comment]
@divshacker @Anantha-Rao12  Have you tried any of those [suggestions on stackoverflow](https://stackoverflow.com/questions/13824669/how-do-you-write-a-link-containing-a-closing-bracket-in-markdown-syntax) ... the problem is with the brackets in the wiki link. It's interesting that locally it is rendered just right.
